tests/com/goide/inspections/GoHighlightingTest.java

```diff
@@ -357,7 +358,7 @@ public void testCGOImportInNonTestFile() {
     myFixture.configureByText("a.go", "package a; import \"C\"");
     myFixture.checkHighlighting();
   }
-  
```

Comment:
Just an advice: please try not to change unnecessary lines – you PR looks dirty after that.